### PR TITLE
Prepare the library to be released as a npm package

### PR DESCRIPTION
**Pull request by manicki**
Requires releases of dependency libraries as npm packages (see: https://github.com/wmde/DataValuesJavaScript/pull/129 and https://github.com/wmde/WikibaseDataModelJavaScript/pull/82).

Includes a commit running Qunit tests using npm test initially submitted as https://github.com/wmde/WikibaseSerializationJavaScript/pull/45.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-			"name": "Adrian Lang",
-			"email": "adrian.lang@wikimedia.de"
-		}
-	],
-	"support": {
-		"issues": "https://phabricator.wikimedia.org/",
-		"irc": "irc://irc.freenode.net/wikidata"
-	},
-	"require": {
-		"data-values/javascript": "~0.8.0|~0.7.0",
```
Comment (thiemowmde) [inline composer.json]: Uh, we never made it compatible with DataValues JavaScript 0.9?

File: package.json [hunk 2]
```diff
@@ -1,13 +1,48 @@
 {
+  "name": "wikibase-serialization",
```
Comment (thiemowmde) [inline package.json]: Again, tab-indention should be the norm for .json files. But this is for later.

File: src/__namespace.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 /**
  * @ignore
  */
+window.wikibase = window.wikibase || {};
```
Comment (thiemowmde) [inline src/__namespace.js]: Why does this use `window.`, but the line below does not? In my opinion this a) should be consistent, and b) avoid `window.` when not really necessary (e.g. certain headless JS engines might need it).

Comment (thiemowmde) [reply src/__namespace.js]: Ping @manicki.

Comment (manicki) [reply src/__namespace.js]: It was needed in this line exactly to have phantom js understand where wikibase object is defined.
It was not apparently needed elsewhere. The whole weirdness is probably going to be irrelevant once this lib uses some proper way of registering its stuff (module.exports or whatever you do in JS this month).

Comment (thiemowmde) [reply src/__namespace.js]: Ok, thanks!

**Pull request merged**